vernac/record.ml

```diff
@@ -640,7 +640,8 @@ let build_class_constant ~univs ~rdata field implfs params paramimpls coers bind
   let cref = GlobRef.ConstRef cst in
   Impargs.declare_manual_implicits false cref paramimpls;
   Impargs.declare_manual_implicits false (GlobRef.ConstRef proj_cst) (List.hd implfs);
-  Classes.set_typeclass_transparency (Tacred.EvalConstRef cst) false false;
+  Classes.set_typeclass_transparency ~locality:Goptions.OptLocal
```

Comment:
shouldn't it be global? also the one below 
(previously false = not local)

Reply:
OK, that has confused me

Reply:
When I change it to global or export, it complains that it is not allowed in sections.